spec/function.dd

```diff
@@ -1542,6 +1542,33 @@ class C
         $(P Template functions, auto functions, nested functions and lambdas can deduce
         the `return scope` attribute.)
 
+$(H3 $(LNAME2 ref-return-scope-parameters, Ref Return Scope Parameters))
+
+        $(P Parameters marked as `ref return scope` come in two forms:)
+
+---
+U xerxes(ref return scope V v);      // (1) ref and return scope
+ref U xerxes(ref return scope V v);  // (2) return ref and scope
```

Comment:
This is still a bit cryptic to me, could you add some (il)legal uses/definitions please?

Reply:
done